src/Resources/translations/messages.fr.yml

```diff
@@ -6,3 +6,8 @@ payplug_sylius_payplug_plugin:
         retrieve_secret_key_in_api_configuration_portal: Retrouvez vos Secret Key LIVE et TEST dans votre <a href="https://portal.payplug.com/#/configuration/api" target="_blank">portail PayPlug</a>
         test_mode: Mode TEST
         payplug_id: 'ID PayPlug : %paymentId%'
+        oney_gateway_label: Oney By PayPlug
```

Comment:
Missing EN translation !

Reply:
After check in specifications, they will be completed when all taks will be done